CombinatorialGames/Game/Special.lean

```diff
@@ -130,4 +130,27 @@ theorem star_fuzzy_down : star ‖ down := by
   rw [← neg_fuzzy_neg_iff, neg_down, neg_star]
   exact star_fuzzy_up
 
+section TinyMiny
+
+variable (x : PGame)
+
+/-! ## Tiny and Miny -/
+
+/-- A tiny game ⧾ is defined as {0 | {0 | -G}}, and is amongst the smallest of the infinitesimals. -/
```

Comment:
(For some reason that I can't imagine), ⧾ and ⧿ are unicode symbols that are designed *specifically* for combinatorial game theory. Should we use these?

Reply:
If we have the tools, might as well use them!

Unfortunately there aren't any keybinds for them in the VSCode Lean extension. Is there anything we could come up with that would be unlikely to clash with anything else?

Reply:
tiny/teenie and miny/meenie work good: if tiny a/o miny or taken, we can use teenie or meenie.